src/behat/CentreonContext.php

```diff
@@ -1223,4 +1223,41 @@ function () use ($currentPage, $expectedProperties)  {
             60
         );
     }
+
+    /**
+     * @param string $hostName the host name to check
+     * @param string $serviceName the service name to check (column description)
+     * @return void
+     * @throws \Exception
+     */
+    protected function waitServiceIsMonitored(string $hostName, string $serviceName): void
+    {
+        $this->spin(
+            function ($context) use ($hostName, $serviceName) {
+                $monitored = false;
+
+                $storageDb = $context->getStorageDatabase();
+                $res = $storageDb->prepare(
+                    <<<'SQL'
+                        SELECT s.service_id
+                        FROM hosts h, services s
+                        WHERE s.host_id = h.host_id
+                        AND h.name = :host_name
+                        AND s.description = :service_description
+                        SQL
+                );
+                $res->bindValue(':host_name', $hostName, PDO::PARAM_STR);
+                $res->bindValue(':service_description', $serviceName, PDO::PARAM_STR);
+                $res->execute();
```

Comment:
The bindValue() is useless because the type is string and execute do a bindValue() with string type by default 
```suggestion
                $res->execute([
                    'host_name' => $hostName,
                    'service_description' => $serviceName
                ]);
```